Source/ARTDataQuery.m

```diff
@@ -54,7 +54,7 @@ - (NSMutableArray *)asQueryItems {
     NSMutableArray *items = [super asQueryItems];
     if (self.untilAttach) {
         NSAssert(self.realtimeChannel, @"ARTRealtimeHistoryQuery used from outside ARTRealtimeChannel.history");
-        if (self.realtimeChannel.state != ARTRealtimeChannelAttached) {
+        if (self.realtimeChannel.state != ARTChannelStateAttached) {
```

Comment:
Well, I changed it because I stated from the IDL that there's the `ChannelType` and the `ChannelState` and I wanted to stay consistent with that. It seems wrong to me having an `ARTChannelEvent` and an `ARTRelatimeChanneState`.